dnf-plugins-core.spec

```diff
@@ -167,6 +167,7 @@ Additionally provides generate_completion_cache passive plugin.
 %package -n %{yum_utils_subpackage_name}
 %if "%{yum_utils_subpackage_name}" == "dnf-utils"
 Conflicts:      yum-utils < 1.1.31-513
+Conflicts:      yum-plugin-copr < 1.1.32
```

Comment:
Change this to `Conflicts:      yum-plugin-copr < 1.1.31-520`

Reply:
Any minor release of that plugin will result that I have to create new patch. We have to always think about future and not only about present situation. I know that there will be not any yum-utils < 1.1.32. I also know that dnf-utils will conflict with yum-utils. But for this time I am going to change it.